app/assets/javascripts/discourse/controllers/history.js.es6

```diff
@@ -252,8 +252,15 @@ export default Ember.Controller.extend(ModalFunctionality, {
     if (viewMode === "side_by_side_markdown") {
       return html;
     } else {
-      const whiteLister = new WhiteLister({ features: { editHistory: true } });
+      const tableTags = ["table", "thead", "tbody", "th", "tr", "td"];
```

Comment:
yeah this is not sitting right ... we have our default markdown whitelister ... why can we not use it.

Reply:
https://github.com/discourse/discourse/blob/b54ba4c952d79d56742e76081d692f1281e88512/app/assets/javascripts/pretty-text/engines/discourse-markdown/table.js.es6#L31-L39

Reply:
so can't we just ask pretty text for the whitelister it uses ? and simply reuse that?

Reply:
I've tried using the same whitelister that pretty-text uses and it did fix this particular issue, however, it introduced a whole new bunch of issues. Mainly it messed up the layout and diff coloring (green and red) because all `diff-ins` and `dif-del` classes were stripped from diff HTML.

All this pain is caused by this bit here:

https://github.com/discourse/discourse/blob/9f558d7e74118d8dc0cb1db04f2b7200f5a3618c/lib/discourse_diff.rb#L248-L253

Specifically, Nokogiri Parser. It seems to be decoding HTML entities when parsing our HTML and returning HTML that's not absolutely safe and needs to be sanitized again.

Looks like there is a way to disable HTML decoding, but it doesn't work :/ (see [this issue](https://github.com/sparklemotion/nokogiri/issues/1284)).

Reply:
re: `diff-ins and dif-del` that is fine, I would then take the whitelister, clone it and add the 2 new classes you need. I follow about nokogiri making trouble.

Reply:
Just to be sure before I start working on this, I'll need to whitelist those two classes with every HTML tag we allow in posts (I don't think I'll have to list all the tags manually in an array, I think I can get them from existing code). Is that OK?